<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>info</class>
 <widget class="QDialog" name="info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
</string>
   </property>
   <property name="text">
    <string>医生信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>医生姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>120</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>所在科室</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>150</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>坐诊时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>170</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>从业经验</string>
   </property>
  </widget>
  <widget class="QComboBox" name="nameComboBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sexLineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="roomLineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="timeLineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="experienceTextEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>50</y>
     <width>91</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>职位</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="jobLineEdit">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>170</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="enterButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>进入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="escButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
